### PR TITLE
feat: add news/blog section

### PR DESCRIPTION
**Pull request by Asmit2952**
## News/blog section desktop

Screenshot (365)

## News/blog Section mobile

Screenshot (368)1

This currently works by fetching data from JSON file. We can highlight important blog/news in this section which won't be updated constantly, something like "What Conda is about?", "How to set up Conda", "How to contribute to Conda" etc. This way user can immediately jump on the important topics.

### CODE REVIEW

**Comment by Asmit2952** [comment]
Hey @jaimergp I think there's some issue with Netify's deployment. The site is building without errors, but latest section is not reflecting on the site

**Comment by jaimergp** [comment]
Check the logs:

```
8:28:37 AM: Module not found: Error: Can't resolve '../components/Header' in '/opt/build/repo/src/pages'
Module not found: Error: Can't resolve '../components/Features' in '/opt/build/repo/src/pages'
[ERROR] Client bundle compiled with errors therefore further build is impossible.
```

If you can't run `npm run build` locally it won't build on Netlify either.

**Comment by Asmit2952** [comment]
> Check the logs:
> 
> ```
> 8:28:37 AM: Module not found: Error: Can't resolve '../components/Header' in '/opt/build/repo/src/pages'
> Module not found: Error: Can't resolve '../components/Features' in '/opt/build/repo/src/pages'
> [ERROR] Client bundle compiled with errors therefore further build is impossible.
> ```
> 
> If you can't run `npm run build` locally it won't build on Netlify either.

`npm run build` runs locally perfectly and builds the website correctly. But when I push the changes, it fails on netlify

**Comment by jaimergp** [comment]
Looks cool and in-brand! Thanks!

One thing I am worried about is whether we are over using the green gradient element.

What if we only "greenify" the highlighted (first) news item, and the others have a secondary color (e.g. light grey). The ones marked with a yellow X, I mean:

image

And another thing, do people usually homogenise the border radius of the rounded corners? Right now, the ones in the news items panels seem a bit too big compared to the other ones present above in the front page. 

Let me know what you think!

**Comment by Asmit2952** [comment]
Yeah we can reduce the use of green in these cards. Another idea that I had is, we can change the color of the card on hover, this will make it more interactive.

And about the border radius, keeping it consistent seems good to me. Let me know, which radius suits more. I'm inclined towards the news cards

File: src/components/News/index.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    )
+  })
+
+  return (
+    <div className={[styles.header, styles.section_padding].join(' ')} id="blog">
+      <div className={styles.news_heading}>
+        <h1 className={styles.gradient_text}>Discover the latest trends with our Conda blog updates</h1>
+      </div>
+      <div className={styles.news_container}>
+        <div className={styles.news_container_groupA}>
```
[the remark this answers is not in the document]
Comment (Asmit2952) [reply src/components/News/index.jsx:27]: Yupp sure

**Pull request merged**